etc/services-camel.properties

```diff
@@ -57,6 +60,7 @@ input.stream=direct-vm:index.start
 
 # The camel URI for handling reindexing events.
 triplestore.reindex.stream=activemq:queue:triplestore.reindex
```

Comment:
should this uri be deleted in favor of the one you added?

Reply:
Its a bit confusing, but they are separate routes. `cdr.triplesupdate.stream.camel` is where requests for reindexing actions comes from (from our clients). `triplestore.reindex.stream` is the internal route in the reindexing router that crawls the hierarchy and triggers indexing of objects.

Reply:
oh, ok, thanks for explaining! would it be worth putting a comment to this effect in the properties file?